build-in-container/run-container.sh

```diff
@@ -17,15 +17,15 @@ run_build_container() {
         ${mount_pts} \
         --privileged \
         --cap-add SYS_ADMIN \
-        mcr.microsoft.com/mariner-container-build:2.0 /mariner/scripts/build-mariner.sh
+        mcr.microsoft.com/mariner-container-build:2.0 /mariner/scripts/build-mariner.sh $container_args
 }
 
 run_interactive_container() {
     docker run \
         ${mount_pts} \
         --privileged \
         --cap-add SYS_ADMIN \
-        -it mcr.microsoft.com/mariner-container-build:2.0 /bin/bash
+        -it mcr.microsoft.com/mariner-container-build:2.0 /mariner/scripts/build-mariner.sh $container_args
```

Comment:
Curious why this is changing from a bash shell to the `build-mariner.sh` script?

Reply:
I refactored this part because now we have multiple optional arguments, which need to be passed to a starter script inside the container. It's much cleaner now, and would be easier to work with as we do more development